test/integration/hooks/deadlines.js

```diff
@@ -0,0 +1,126 @@
+const request = require('supertest');
+const assert = require('assert');
+const moment = require('moment-business-time');
+const workflowHelper = require('../../helpers/workflow');
+const { holc } = require('../../data/profiles');
+const ids = require('../../data/ids');
+
+describe('Project deadlines', () => {
+  before(() => {
+    return workflowHelper.create()
+      .then(workflow => {
+        this.workflow = workflow;
+      });
+  });
+
+  beforeEach(() => {
+    return Promise.resolve()
+      .then(() => this.workflow.resetDBs())
+      .then(() => this.workflow.clearTaskList());
```

Comment:
if `resetDBs()` calls `tfdb.reset()`, why the second call to `clearTaskList()`?

Reply:
Removed.